app/assets/javascripts/discourse/tests/acceptance/composer-test.js

```diff
@@ -1360,3 +1360,29 @@ acceptance("Composer - default category not set", function (needs) {
   });
 });
 // END: Default Composer Category tests
+
+acceptance("Composer - current time", function (needs) {
+  needs.user();
+
+  test("composer insert current time shortcut", async function (assert) {
+    await visit("/t/internationalization-localization/280");
+
+    await click("#topic-footer-buttons .btn.create");
+    assert.ok(exists(".d-editor-input"), "the composer input is visible");
+    await fillIn(".d-editor-input", "and the time now is: ");
+
+    await triggerKeyEvent(".d-editor-input", "keydown", ".", {
+      shiftKey: true,
+      metaKey: true,
+    });
+
+    const time = moment().format("HH:mm:ss");
+    const date = moment().format("YYYY-MM-DD");
+
+    assert.strictEqual(
+      query("#reply-control .d-editor-input").value.trim(),
+      `and the time now is: [date=${date} time=${time} timezone="Australia/Brisbane"]`,
```

Comment:
I suspect this can be flakey. I tried using the `fakeTime` helper, but that was running into timezone issues and/or timeouts. 

Curious if you have any ideas @jjaffeux.

Reply:
Might be wrong but given how it's done, chances it would be flakey are low I would think. Let's see if it causes issues before going with more complex solutions.

Reply:
Sure, sounds good.